excalidraw-app/vite.config.mts

```diff
@@ -5,7 +5,7 @@ import { ViteEjsPlugin } from "vite-plugin-ejs";
 import { VitePWA } from "vite-plugin-pwa";
 import checker from "vite-plugin-checker";
 import { createHtmlPlugin } from "vite-plugin-html";
-import { woff2BrowserPlugin } from "../scripts/woff2/woff2-vite-plugins";
+import { woff2BrowserPlugin } from "./scripts/woff2-vite-plugins";
```

Comment:
I wanted to keep `woff2-vite-plugin` alongside with `woff2-esbuild-plugin`, as those two are essentially the same thing, just one configured for `excalidraw-app` build with vite+rollup and other for packages like `@excalidraw/excalidraw` and `@excalidraw/utils` build with pure esbuild. Thus keeping both on root.

Could we still keep both together? Notice that the current location in script for vite & esbuild build plugins also isn't ideal (they don't really belong to `scripts`).

<img width="306" alt="image" src="https://github.com/user-attachments/assets/cb7c42ea-5e07-44c5-9dd5-f53bbdd6f790">

Reply:
This is the reason the Docker build failed, i.e. the yarn --cwd ./excalidraw-app build command cannot load the script from outside the working directory. I would have to investigate if this could be circumvented somehow, but it would be much easier to host scripts related to apps in the app directory.